static/templates/keyboard_shortcuts.hbs

```diff
@@ -1,59 +1,59 @@
-<div class="overlay-modal" id="keyboard-shortcuts" tabindex="-1" role="dialog"
-  aria-label="{{ _('Keyboard shortcuts') }}">
+<div class="overlay-modal hide" id="keyboard-shortcuts" tabindex="-1" role="dialog"
```

Comment:
Why did you add the `hide` class here when migrating this?

Reply:
I suspect doing so is in error; this tab is the default tab for the widget, and thus I think it's intentional that it doesn't have that class.  (It's also possible that we could remove that class from all 3 widgets).  So I removed this before merging.

Reply:
Oh, Right! Thanks :+1: